server/views/case-summary.njk

```diff
@@ -38,8 +38,14 @@
 {% block caseContent %}
 
     <div class="govuk-grid-row">
-        <div class="govuk-grid-column-full">
+        <div class="govuk-grid-column-three-quarters">
             <h1 class="govuk-heading-l">Case summary</h1>
+
+        </div>
+        <div class="govuk-grid-column-one-quarter">
+            {% if enableCaseHistory %}
+                <a class="moj-primary-navigation__link" href="{{ caseHistoryUrl }}">View case history (&#946;)</a>
```

Comment:
I had to remove the usage of the toggle flag too which disables the route as well

Reply:
Maybe better to use the `class="govuk-visually-hidden"`?

Reply:
@sarahyjja that would cause screen readers to continue to read it out which we don't want. For now that link is purely a convenience for the PIC team to give us the ability to review and understand a case history before we do the full implementation. We don't want users to know about it yet.